_guides/topic/community.adoc

```diff
@@ -5,33 +5,34 @@ permalink: /community/
 include::../attributes.adoc[]
 = {project-name} - Community
 
-== An Open Project
+== Getting help
 
-{project-name} is an Open Source project licensed under the https://www.apache.org/licenses/LICENSE-2.0[Apache License version 2.0]. First and foremost, it is an open community where contributions, ideas and discussions are done in the open and welcome contributors. Let's join forces in building the future of Java applications.
+=== Documentation
 
-== Code and issue tracker
+We have a lot of documentation for getting started with Quarkus, be sure to link:{quarkus-site-getting-started}[check it out]! +
```

Comment:
I'm just wondering if we should also link the Guides page here. I'll merge this one, feel free to open a new one if you think it's a good idea.